### ISSUE TITLE
Use MDN's browser-compat-data to power the site

### ISSUE THREAD

**Issue by connorshea** [mozilla/platform-status#534]
The MDN [browser-compat-data project](https://github.com/mdn/browser-compat-data) is at a point now where almost all of MDN's compatibility tables have been moved over to it.

I think it'd be a good idea to change the Platform Status site such that the data for support in browsers other than Firefox is sourced from the BCD package.

Using the BCD package would do a few things:

- The Platform Status site would have more accurate, up-to-date data on external browser support
- This would help to centralize the data in a single place for multiple browser vendors to contribute
- The data could be more granular, e.g. including the specific browser version a given feature was implemented in, or whether the browser implements all of the subfeatures for a given API.

For one example of data that would be significantly improved, right now the [Abort API](https://platform-status.mozilla.org/#abort-api) has no info for any browser other than Firefox, but the [MDN article's compatibility table](https://developer.mozilla.org/en-US/docs/Web/API/AbortController#Browser_compatibility), powered by the browser-compat-data package, has detailed info on every major browser.

On the Platform Status site:
<img width="1016" alt="screen shot 2018-07-30 at 5 50 09 pm" src="https://user-images.githubusercontent.com/2977353/43429520-14e73a34-9421-11e8-9e88-8f4493669277.png">

On the MDN article:
<img width="1068" alt="screen shot 2018-07-30 at 5 50 14 pm" src="https://user-images.githubusercontent.com/2977353/43429521-150b4fd2-9421-11e8-8466-fdbdc056e247.png">

**Comment by stof** [comment]
Getting data only from BCD might not work, as it won't have the distinction between "no signals", "considering" or "in development" (as far as BCD is concerned, all these are "no support"). But it might indeed be good as an input, to at least mark any supported feature as shipped.

**Comment by connorshea** [comment]
Yup, that was what I was thinking. Its not very useful for signaling necessarily but it is useful for features that have been implemented.
